extension/test/inputs/HelloWorld/HelloWorld.cxml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<ComponentViewModelData xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="HelloWorld" Version="2">
+<ComponentData xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="HelloWorld" Version="2">
```

Comment:
This PR is for xcomponent V5 only ?
So this PR has to be merged on a v5 branch (xcomp v5 is not released yet)